packages/common/src/HardhatConfig.ts

```diff
@@ -70,13 +72,30 @@ export function getHardhatConfig(
         testBlacklist,
       },
       localhost: { url: "http://127.0.0.1:9545", timeout: 1800000, testBlacklist },
+      mainnet: { chainId: 1, url: getNodeUrl("mainnet", true), accounts: { mnemonic } },
```

Comment:
orded by chain ID now

Reply:
Don't you need `Legacy_Testable` and `Legacy_Lockable` too?

Reply:
I think that these are compiled indirectly due to the inheritance path. I aded them anyways